docs/bs-seq-primer.md

```diff
@@ -0,0 +1,127 @@
+
+Mapping bisulfite-treated sequences to a reference genome presents several computational challenges:
+
+1. **Reduced DNA Complexity**:
+
+   - Due to the conversion of cytosines to thymines, the DNA code becomes less diverse, increasing the likelihood of ambiguous alignments
+
+2. **Multiple DNA Strands**:
+
+   - The presence of four possible DNA strands (and their combinations) for each genomic locus increases the alignment search space
+
+3. **Variable Methylation States**:
+   - Each read can theoretically represent any possible methylation state for a locus, further complicating the alignment process
+
+## **Applications of Bisulfite Sequencing**
+
+Bisulfite sequencing is employed to study:
+
+- **Genome-wide DNA methylation patterns**: Understanding epigenetic regulation of genes
+- **Differential methylation**: Investigating differences between different samples (e.g. healthy versus diseased states)
+- **Epigenetic inheritance**: Studying methylation changes across generations
+
+---
+
+## Three-Base Aligners: Bismark and BWA-Meth
+
+Bisulfite sequencing converts unmethylated cytosines into uracils (later read as thymines during PCR), leading to a DNA code represented predominantly by three bases (A, G, and T).
+
+Aligning these three-base reads against a standard reference genome is non-trivial, as the original strand identity and methylation state are obscured by the conversion.
+
+To address these challenges, specialized “three-base aligners” have been developed to accurately map bisulfite-treated reads and infer their original strand context and methylation status. Here, we primarily highlight the aligners used in the nf-core/methylseq pipeline.
+
+### Bismark ([docs](https://felixkrueger.github.io/Bismark/); [publication](https://pmc.ncbi.nlm.nih.gov/articles/PMC3102221/))
+
+- Bismark resolves strand ambiguity by performing up to four parallel alignments
+
+- First, sequencing reads are converted _in-silico_ to represent both forward and reverse strand conversions (C-to-T and G-to-A), mirroring fully bisulfite-converted versions of the reference genome
+
+- Each read set is then aligned using Bowtie2 (alternatively HISAT2 or minimap2) against equally converted references; this enables support for indels, local alignments, and bisulfite converted RNA-seq-type or long reads (e.g. EM-seq using Nanopore or Pac Bio reads)
+
+- By comparing these (up-to) four alignments, Bismark identifies each read’s correct strand origin
+
+This approach allows Bismark to handle directional, PBAT, amplicon, and non-directional libraries robustly and to accurately align reads that represent partially methylated cytosines.
+
+### BWA-Meth ([docs](https://github.com/brentp/bwa-meth); [publication](https://arxiv.org/abs/1401.1129))
+
+- BWA-Meth adapts the BWA-MEM algorithm for bisulfite data, providing efficient, flexible alignments with support for indels, local alignments, and streaming-based workflows
```

Comment:
In what way exactly, what's the technical diff with bismark, given you gave the very nice description above?